lexer/lexer.go

```diff
@@ -7,15 +7,22 @@ import (
 )
 
 type Lexer struct {
-	input string
-	pos   int
+	input    string
+	pos      int
+	lineMode bool
 }
 
+// Mode with input expected the be complete (multiline/file).
 func New(input string) *Lexer {
 	l := &Lexer{input: input}
 	return l
 }
```

Comment:
```suggestion
 func New(input string) *Lexer {
 	return &Lexer{input: input}
 }
```

Reply:
fixed in https://github.com/grol-io/grol/commit/5b28ec0cef422e46b5ba9867ffb7726e18b4c192
was meant to be an PR but I need to fix the permissions

Reply:
> not sure it's how it is, must have had something else 

As far as I know, only simple types like int, string need a variable.

https://github.com/golang/go/issues/45624